### ISSUE TITLE
Trouble with Composer part of install 

### ISSUE THREAD

**Issue by crsdtech** [snipe/snipe-it#2159]
I've followed the installation directions to the letter but I'm hung at the Install Dependencies step: "Right click and select Composer Install. This will install the dependencies." What do I right click on? 

I have a similar issue with the CLI Installer part of the documentation: "Right Click and select Use Composer here." When I right click in open space in the c:\inetpub\wwwroot\snipe-it folder, I don't see an option for this.

- Snipe v2.1.2
- Composer v1.1.2
- Windows Server 2012 R2 Standard, IIS 8.5

**Comment by mmcleod82** [comment]
try opening that directory at a command prompt and typing `composer install`

**Comment by crsdtech** [comment]
Running the command `composer install` returned problems: "`Problem 1 - intervention/image 2.3 requires ext-fileinfo * ->`"..."`Problem 2`"...

I added `extension=php_fileinfo.dll` to my php.ini file (it was missing, not just commented out) and tried `composer install` again and it installed!

While still in that directory I ran the `php artisan app:install --env=production` command and the prompts were presented.

Thank you, mmcleod, for such a fast response! Hopefully this helps someone else out in the future.
